Readme.md

```diff
@@ -95,6 +95,8 @@ We evaluated the CompGCN repository with the following configurations.
   # CompGCN (Composition: Circular Correlation)
   python run.py -score_func conve -opn corr
   
+  python run.py -score_func ctkgc -init_dim 100 -embed_dim 100 -opn sub -data codex_s
```

Comment:
Should we think about adding ranges to parameters given or are they fixed?

Reply:
And the dataset is called codex-s with a hyphen:
```suggestion
  python run.py -score_func ctkgc -init_dim 100 -embed_dim 100 -opn sub -data codex-s
```